### PR TITLE
Refactor/error logging

### PR DESCRIPTION
**Pull request by axe312ger**
* errors are logged to error log file with extended information for manual debugging
* single errors do not stop the import anymore
* errors are properly formatted in one line for the CLI output
* npmlog dependency removed for proper output in CLI
* ~now aborts infinite publishing queues~ (queue feature was removed in another PR in favor of reference sorting)

### CODE REVIEW

File: lib/push/creation.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ export function createEntities (context, entities, destinationEntities) {
       .then(partial(creationSuccessNotifier, operation))
       .catch(partial(handleCreationErrors, entity))
   }, {concurrency: 6})
+  .then((entries) => entries.filter((entry) => entry))
```
Comment (Khaledgarbaya) [inline lib/push/creation.js]: any reason for this `filter` ?

Comment (axe312ger) [inline lib/push/creation.js]: Publishing does not handly empty entities. The `createEntries` function does also filter the output

**Comment by axe312ger** [comment]
@Khaledgarbaya I added some minor changes I came across while implementing it in `contentful-import`

The new error processing code was faulty, plus I found some more code we do not use at all.

**Comment by axe312ger** [comment]
closed in favor of #126

**Pull request closed without merging**